src/main/resources/db/migration/V009__structuredQuery.sql

```diff
@@ -0,0 +1 @@
+ALTER TYPE INQUIRY_CRITERIA_TYPE ADD VALUE 'IC_SQ';
```

Comment:
Can we use IC_STRUCTURED_QUERY instead? I don't know if we know that SQ is in 6 month. Also This needs a newline at the end of the file.